index.html

```diff
@@ -345,6 +358,15 @@ <h3>MPEG Working Draft on Carriage of Web Resources in ISO BMFF</h3>
           including overlays.
         </p>
       </section>
+      <section>
+        <h3>WebVTT Cues</h3>
+        <p>
+          [[WEBVTT]] is a W3C specification that provides a format for web video
+          text tracks.  The <code>WebVTT Cue</code> is a text track cue that provides
+          attributes can be used in rendering of the cue text in the context of a web page.
```

Comment:
the grammar looks wrong here. In the next few minutes I'll make a PR to try and tweak it.